src/expectType.ts

```diff
@@ -7,144 +7,71 @@
  *
  */
 
+import type expect from 'expect';
+import type { ExpectedAssertionsErrors } from 'expect/build/types';
+
 export declare type AsymmetricMatcher = Record<string, any>;
+
 export declare type Expect = {
-    <T = unknown>(actual: T): Matchers<T>;
+  <T = unknown>(actual: T): folio.Matchers<T>;
+
+  // Sourced from node_modules/expect/build/types.d.ts
+  addSnapshotSerializer(arg0: any): void;
```

Comment:
I think some of these methods won't work in Folio, because they rely on jest integration. Let's go over them and only leave the safe ones?

Reply:
Only the first one I saw was a stub, the others should work as expected. :+1: